res/database/Default/user/jorgaeff.xml

```diff
@@ -273,7 +273,6 @@
 		</news>
 
 		<news id="news-jorgaeff-1987-0-UdSSR-01b" type="2" thread_id="news-jorgaeff-1987-0-UdSSR-01" creator="8936" created_by="jorgaeff" comment="TODO: ref persons Michailow, Markus Trust">
-			<availability script="TIME_YEAR=1987" />
```

Comment:
Leave unchanged

Reply:
isn't that redundant when the previous entry already has this parameter?

and if the previous item happens to be broadcasted in the last moment of 1987, wouldn't that mean that this one would be ignored?

Reply:
Indeed, that's why I wanted the parallel threads. (Leave the two threads as independent of each other as possible.)

Reply:
I don't quite understand that reasoning, but I'm undoing the deletion.